tests/landIce/FO_GIS/input_fo_gis_coupledT_muelu_kokkos.yaml

```diff
@@ -0,0 +1,353 @@
+%YAML 1.1
+---
+ANONYMOUS:
+  Build Type: Tpetra
+  Debug Output:
+    Write Solution to MatrixMarket: 0
+  Problem:
+    Phalanx Graph Visualization Detail: 1
+    Solution Method: Steady
+    Name: LandIce Coupled FO H 3D
```

Comment:
This one is currently not tested, should you remove it for now? I'm not sure if we've tried coupled with muelu even on CPU.